adr/2021_06_15_api_platform_config_customization.md

```diff
@@ -0,0 +1,42 @@
+# API - Customization
+
+* Status: accepted
+* Date: 2021-06-15
+
+## Context and Problem Statement
+
+Its difficult to overwrite or remove endpoints.
+
+## Considered Options
+
+### Overwriting resource configs
+
+For now, only way to customize api resources is by overwriting every configuration associated to the endpoint.
+
+* Bad, because its not easy to configure.
+* Bad, because it requires to change many files in end application.
```

Comment:
* Bad, because any changes in the new Sylius version will not be applicable to overwritten configs